src/pages/ProductListing/components/FilterContainer/FilterContainer.jsx

```diff
@@ -0,0 +1,53 @@
+// All local imports
+import filterStyles from "./FilterContainer.module.css";
+import { filtersData, ratingFilter } from "../../../../data";
+import { CheckboxFilterType } from "../FilterTypes/CheckboxFilterType";
+import { RatingFilterType } from "../FilterTypes/RatingFilterType";
+
+export function FilterContainer() {
+    return (
+        <aside
+            aria-label="Filter options"
+            className={filterStyles["filter-container"]}
```

Comment:
What's the advantage of using className the way you have any and why not simply use as className="classname" ?